scripts/rust-utils/uniprot-parser-tryptic/src/main.rs

```diff
@@ -3,7 +3,7 @@ use clap::Parser;
 use dat_parser::uniprot_dat_parser;
 use std::path::PathBuf;
 use tables_generator::models::Entry;
-use tables_generator::table_writer::{EntryTableWriter, PeptideTableWriter};
+use tables_generator::table_writer::{EntryTableWriter, PeptideTableWriter, ProteomeTableWriter};
```

Comment:
This can be removed from the tryptic parser. Since we'll never need it there, I think it's best we also don't implement it there (to minimize code duplication between the different executables).